### DISCUSSION TITLE
玲珑相关的文档需要整理和更新

### DISCUSSION THREAD

**Discussion by black-desk** [linuxdeepin/developer-center#3569] [Features Request & Ideas | 特性请求 & 头脑风暴]
1. 玲珑相关的文档中存在一些需要更新的地方
  有用户尝试在其他发行版上编译并使用玲珑相关的工具, 在该用户提出的[问题](https://github.com/orgs/linuxdeepin/discussions/3567)中我们发现玲珑相关的文档中存在一些需要更新和详细说明的地方.
2. 目前玲珑的文档有部分未公开, 内容分散在两个代码仓库中, 需要进行整理.
3. 目前玲珑的项目主页已经公开, 但其内容的更新与github工作流结合得不紧密. 无法做到仓库更新后即时生效. 建议迁移到github pages.

ping @myml @kamiyadm @zccrs @Iceyer

**Comment by myml** [comment]
我来解决第三个问题
- [x] 使用github action构建网站 https://github.com/linuxdeepin/linglong-homepage/pull/3
- [x] 支持部署到github pages https://github.com/linuxdeepin/linglong-homepage/pull/3
- [ ] 申请域名linglong.dev代理到github pages

**Reply by Iceyer** [reply]
需要的域名配置可以私发给我处理
